pytest_logger/plugin.py

```diff
@@ -13,6 +14,28 @@
 else:
     string_type = str
 
+os_symlink = getattr(os, "symlink", None)
+if callable(os_symlink) is False and platform.system() == 'Windows':
+    
+    def symlink_win(source, linkname):
+        """ Symlink implementation on Windows platform, working from Vista
+        inspired by http://stackoverflow.com/questions/6260149/os-symlink-support-in-windows
+        """
+        import ctypes
+        flags = 1 if source is not None and os.path.isdir(source) else 0
+        try:
+            linkname = linkname.replace('/', '\\')
+            csl = ctypes.windll.kernel32.CreateSymbolicLinkW
```

Comment:
I understand (from win32com.client code) that first link uses COM API to create shortcut, which is a text file with filepath, not actually a symlink.

Second link says that symlinks require special privileges. It seems python3 does it almost identically to proposed code:
https://github.com/python/cpython/blob/master/Modules/posixmodule.c#L7221
using CreateSymbolicLinkW, which requires SE_CREATE_SYMBOLIC_LINK_NAME privilege.

Let's go with the solution you proposed.